pipeline-steps/aio_prepare.groovy

```diff
@@ -18,27 +18,9 @@ def prepare(){
           } //env
         } //color
       } //dir
-      withCredentials([
-        string(
-          credentialsId: "dev_pubcloud_username",
-          variable: "PUBCLOUD_USERNAME"
-        ),
-        string(
-          credentialsId: "dev_pubcloud_api_key",
-          variable: "PUBCLOUD_API_KEY"
-        ),
-        string(
-          credentialsId: "dev_pubcloud_tenant_id",
-          variable: "PUBCLOUD_TENANT_ID"
-        )
-      ]){
-        dir("/opt/rpc-gating"){
-          git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
-        } //dir
-        dir("/opt/rpc-gating/playbooks"){
-          ansiblePlaybook playbook: "aio_config.yml"
-        } //dir
-      } //withCredentials
+      common.prepareConfigs(
```

Comment:
Noted. I've moved this out of the deployment conditional block. It's not re-pushed just yet; pending a clarification on the other comment thread.

I've also updated the default `STAGES` parameter with the save caveat.